moped-editor/src/App.js

```diff
@@ -51,9 +55,25 @@ const useClient = () => {
       };
     });
 
+    const errorLink = onError(({ graphQLErrors, networkError }) => {
+      if (graphQLErrors) {
+        // Concatenate all error messages to create a single error and message
+        const concatenatedMessage = graphQLErrors
+          .map(({ message }) => message)
+          .join("; ");
+
+        const combinedError = new Error(concatenatedMessage);
+        setError(combinedError);
+      }
+
+      if (networkError) {
+        setError(networkError);
+      }
+    });
+
     return new ApolloClient({
       // Join authLink and httpLink to handle auth in each request
-      link: authLink.concat(httpLink),
+      link: from([errorLink, authLink, httpLink]),
```

Comment:
`from` does the same thing as `.concat`. We just have more than two so we need to use `from` to chain the links together. `errorLink` handles errors, `authLink` handles our auth logic needed to make successful requests, and `httpLink` sets up our connection to the Hasura graphql endpoint.